### ISSUE TITLE
[SF_ACC][1.7.1][ui5-combobox]: Screen reader announcing the Edit combo box collapsed - should announce as "combo box" only

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#5853]
### **Bug Description**
ui5-combobox announcement issue:
When focus on combobox, screen reader announcing "Edit combo box collapsed". Expect announce as "combo box" only.
Here is the text announced by screen reader, pls take a look.
![image](https://user-images.githubusercontent.com/78520316/192676971-e7de1efd-52a1-434c-8997-6c2766142b49.png)

### **Steps to Reproduce**
1. Go to qacand and nav to omp manage my opportunity page -> my postings 
(https://qacand.hcm.ondemand.com/sf/omp?_s.crb=8cElggwPcFRPL%252bBVTeIUwWHSFu8E3s3vCBi%252f4Nw2WXs%253d#/manage?tab=opportunities).
Use Mock data if real api response is slow:  https://qacand.hcm.ondemand.com/sf/omp?mock&_s.crb=8cElggwPcFRPL%252bBVTeIUwWHSFu8E3s3vCBi%252f4Nw2WXs%253d#/manage?tab=opportunities
2. Tab to combobox displayed after filter by label
<img width="1121" alt="Screen Shot 2022-09-28 at 11 07 12" src="https://user-images.githubusercontent.com/78520316/192678613-d90582e6-a504-4840-a7c0-54b3748c1a2c.png">

### **Context**
- UI5 Web Components version: {1.7.1}
- Browser: {Chrome}
- Affected component: {ui5-combobox}

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

Thanks 
Josephine Su

**Comment by niyap** [comment]
Hello @sfsf-xweb-sh,

That is the expected speech output. "Collapsed" has to be announced in order to give the information to the users that the combobox dropdown is closed. When, it is open "expanded" is announced.

Kind Regards,
Niya

**Comment by sfsf-xweb-sh** [comment]
@niyap 
Thanks for your quick response. What about the "Edit" announced before combobox? If it's also expected, I'll ask QAS team to close related defect ticket.
